src/js/stores/AppFormStore.js

```diff
@@ -186,6 +186,36 @@ const resolveAppKeyToFieldIdMap = {
   "healthChecks": "healthChecks"
 };
 
+// Validate all fields in form store and update validationErrorIndices.
+// Return true if all checks passed, otherwise false.
+function checkAllFieldsForValidity(fields) {
+  return Object.keys(fields).reduce((memo, fieldId) => {
```

Comment:
to expand on @aldipower's somewhat laconic response ;) I made this mistake when we first wrote this method. Stopping at the first falsy value means that if there are multiple problems with the app config (which is possible if it's just been manually edited) only the first one will be flagged.